src/styles/_global.sass

```diff
@@ -42,6 +42,18 @@ textarea
 	font-size: inherit
 	line-height: inherit
 
+button
+	border: none
+	background: transparent
+	padding: 0
+	margin: 0
+	outline: none
```

Comment:
https://a11yproject.com/posts/never-remove-css-outlines/

min. přesunout pod .button, který už :focus nějak nastylován má

Reply:
1. Žádná grafika nepoužívá nativní outline.
2. `.button` má nějaký jednoduchý styl na `focus` a `hover`, aby kodér věděl, co všechno má řešit/dostylovat.
3. Bylo to špatně nastavené a outline se stejně zobrazoval. :D (Nový commit fixuje.)